classes/tl_form.php

```diff
@@ -79,7 +80,9 @@ public function generateTokens(array $arrData, array $arrForm, array $arrFiles,
         $arrTokens = array();
         $arrTokens['raw_data'] = '';
         $arrTokens['raw_data_filled'] = '';
+        $arrTokens['template_data'] = [];
 
+        // Data to use in message.
```

Comment:
```suggestion
        // Data to use in message
```
Nitpick: the other comments currently do not end with a period.